app/models/course/assessment/question.rb

```diff
@@ -5,4 +5,17 @@ class Course::Assessment::Question < ActiveRecord::Base
   has_and_belongs_to_many :tags
 
   delegate :to_partial_path, to: :actable
+
+  # Attempts the given question in the submission. This builds a new answer for the current
+  # question.
+  #
+  # @param [Course::Assessment::Submission] submission The submission which the answer should
+  #   belong to.
+  # @return [Course::Assessment::Answer] The answer corresponding to the question. It is required
+  #   that the {Course::Assessment::Answer#question} property be the same as +self+. The result
+  #   should not be persisted.
+  def attempt(submission)
```

Comment:
...probably because you don't understand the design of the assessments. what do you want to rename it to?

student attempts a question in the context of a submission, you get an answer of polymorphic type. you cannot attempt a question without a submission, and you cannot attempt a submission. I don't see any other way to phrase or call this.